src/scss/components/events-highlights.scss

```diff
@@ -94,4 +112,62 @@
       padding: 21px 0;
     }
   }
+
+  @include viewport-desktop() {
+    & {
+      background-position: -3vw;
+      background-position-y: 23vw;
+      background-size: auto 50%;
+      display: flex;
+      flex-direction: row;
+      max-width: 1175px;
+    }
+
+      &.event__feature {
```

Comment:
Could you give a format to this file?

Reply:
![image](https://user-images.githubusercontent.com/65635471/93718521-97b73100-fb42-11ea-8e7c-398af06bbad7.png)
![image](https://user-images.githubusercontent.com/65635471/93718530-ad2c5b00-fb42-11ea-9282-fe571a5014c8.png)
![image](https://user-images.githubusercontent.com/65635471/93718531-b4536900-fb42-11ea-9da1-d37213322f8f.png)

the size of the elements has to be arranged and a background pattern image is missing

Reply:
the size of the elements has to be arranged to what size?, because they are responsive right now, growing according to the viewport